frameworks/digital-outcomes-and-specialists/questions/briefs/technicalCompetenceCriteriaParticipants.yml

```diff
@@ -0,0 +1,28 @@
+name: Technical competence criteria
+question: Technical competence criteria
+question_advice: |
+  Technical competence is how well the supplier can do the work you need them to do.
+
+  Evaluate suppliers’ technical competence using criteria for:
+
+  - essential skills and experience
+  - nice-to-have skills and experience
+  - proposal
+
+  **Scoring (optional)**<br />
+  If you want to score individual criteria, you must add points for everything. Read [how to score individual criteria](https://www.gov.uk/guidance/how-to-set-your-evaluation-criteria-when-buying-digital-outcomes-and-specialists-services#giving-points-to-individual-criteria).
+
+  The points you give for nice-to-have skills and experience must be lower than the points you give to the supplier’s essential skills and experience and proposal.
```

Comment:
Also, why do we have this explanation a different way round here than for specialists?  (i.e. `nice-to-have must be lower than essential` vs `essential must be higher than nice-to-have`) I think it would be nicer to have the same text in both places, as it's exactly the same message.